src/Endpoints/Tasks.php

```diff
@@ -17,12 +19,14 @@ class Tasks extends BaseEndpoint
     /**
      * Retrieve all tasks.
      *
+     * @param DateTime|null $updatedSince
      * @return TasksModel
-     * @TODO: updated_since
      */
-    public function all()
+    public function all(DateTime $updatedSince = null)
     {
-        $response = $this->httpClient->get('/tasks');
+        $uri = '/tasks';
```

Comment:
This does not need to be an extra variable, it can be inlined in the call to `Utils::appendUpdatedSinceToUri`.